spec/cordial/automation_templates_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'spec_helper'
+
+RSpec.describe Cordial::AutomationTemplates do
+  let(:key) { 'promo_test' }
+  let(:name) { 'Promo 2018' }
+  let(:channel) { 'email' }
+  let(:classification) { 'promotional' }
+  let(:base) { 'hourly' }
+  let(:subjectEmail) { 'One Day Only Sale' }
+  let(:from) { 'info@example.com' }
+  let(:reply) { 'info@example.com' }
+  let(:description) { 'Promotions Team' }
+  let(:text) { '<div>Hello World</div>' }
+
+  describe '#create', :vcr do
```

Comment:
It would be amazing if you test both possible scenarios (200 and 422)

Reply:
Doing testing we noticed that code was `201` 👌